HelpSource/Classes/Object.schelp

```diff
@@ -226,6 +238,46 @@ code::
 'worth trying' === 'worth trying'; // identical (symbols are unique)
 ::
 
+The low information of the link::Classes/Symbol:: class instances, returned by the code::.dump:: method (see: link::#-dump::), shows only its class and its value (symbol) because it is unique:
+code::
+'worth trying'.dump
+::
+teletype::
+   Symbol 'worth trying'
+-> worth trying
+::
+
+Also, the link::#-hash:: and link::#-identityHash:: methods return the same emphasis::hash table index:::
+
+code::
+'worth trying'.hash
+::
+teletype::
+-> 1726909008
+::
+
+code::
+'worth trying'.identityHash
+::
+teletype::
+-> 1726909008
+::
+
+code::
+'worth trying'.hash ==  'worth trying'.identityHash
+::
+teletype::
+-> true
+::
+code::
+'worth trying'.hash === 'worth trying'.identityHash
+::
+teletype::
+-> true
+::
+
+For details on the identity of link::Classes/Integer::, link::Classes/Float::, link::Classes/Array:: and link::Classes/String::, see the link::#equality & identity discussion#Discussion section in the ==(obj) method::.
+
```

Comment:
Yes, I knew that when I wrote that part.

.hash` only has the explanation of `.hash`, and `.identityHash` only has the explanation of `.identityHash`.  I thought it would be good to put this related information here, to build a friendly explanation for newbies.

I have reconstructed this part. What do you think?